README.md

```diff
@@ -1,36 +1,7 @@
-# To install:
-Create a python [venv](https://docs.python.org/3/tutorial/venv.html) for this project to keep all the packages separate and at the correct version.  
-`python3 -m venv .venv`  
-To use this newly created venv you must activate it every time you want to start using it.  
-For windows use:  
-`.venv\Scripts\activate.bat`
-For Unix or MacOS use:  
-`source .venv/bin/activate`  
-Update pip to the newest version with the following command:  
-`pip install --upgrade pip`  
-Next install all the requirements for this project by running:  
-`pip install -r requirements.txt`  
-This project uses [MariaDB](https://mariadb.org/download/) as the database, which needs to be installed.
+Website for Cosmos, the international students association at the TU/e.
 
-# Linting
-In order to check the code for errors, we use flake8, which also gives some style enforcement. The configuration for flake8 can be found in the tox.ini file. Additional plugins installed for flake8 give support for django specific linting, pep8 compliant naming and correct sorting of imports. To install flake8 and all plugins run:  
-`pip install flake8 flake8-django pep8-naming flake8-isort`  
-To make sure everything is checked before creating a commit, install pre-commit and install the git hook.  
-`pip install pre-commit`  
-`pre-commit install`  
+# Development
+Information for setting up the project locally can be found on the wiki [here](https://github.com/bcvandendool/Cosmos/wiki/Setting-up-for-local-dev). If you are interested in contributing to the project, please look [here]() for the pull request procedure. If you are interested in joining the web committee, who are responsible for developing and maintaining this website, please contact the chair of the web committee.
```

Comment:
Will contribution be added later?

Reply:
yeah, a lot more docs need to be written still, but they arent the main priority rn untill the mvp is done